app/javascript/src/components/Dashboard/EditOrDeleteActionDropdown.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { MenuHorizontal, MenuVertical } from "neetoicons";
+import { Dropdown } from "neetoui";
+
+const EditOrDeleteActionDropdown = ({
```

Comment:
This name is good. But we can keep it more compact and on point:
```suggestion
const ActionDropdown = ({
```

^ The above name is pretty self-explanatory and conveys what the other name was trying to point out.